simulator/autoware_vehicle_door_simulator/src/dummy_doors.cpp

```diff
@@ -16,10 +16,10 @@
 
 #include <memory>
 
-namespace vehicle_door_simulator
+namespace autoware::simulator::vehicle_door_simulator
```

Comment:
No component name "simulator" needed

Reply:
Thank you, and sorry for my silly mistake in this namespace issue. I'll fix and push soon.

Reply:
Fixed by [this commit](https://github.com/autowarefoundation/autoware.universe/pull/9997/commits/04aeedeb83786fbfa6ad8aeb7a0ae9d064edfbdd)